R/bettr-main.R

```diff
@@ -1,22 +1,46 @@
 #' Launch bettr app to explore and aggregate performance metrics
 #' 
-#' @param df A \code{data.frame} in wide format. 
-#' @param idCol Character scalar, indicating which column of \code{df} 
+#' @param df A \code{data.frame} in wide format. Should contain one column 
+#'   with the IDs of the entities to be compared, and one column for each 
+#'   metric to use for the comparison.
+#' @param idCol Character scalar, indicating the name of the column of \code{df} 
 #'   that contains IDs of the entities to be compared (e.g., methods).
-#' @param metrics_num,metrics_cat Character vectors, indicating which of the 
-#'   columns of \code{df} that correspond to, numeric and categorical, 
-#'   respectively, metrics of interest.
+#' @param metrics Character vector, indicating which of the 
+#'   columns of \code{df} that correspond to metrics of interest. Only metrics
+#'   included here will be displayed.
 #' @param initialWeights Named numeric vector providing initial weights for 
-#'   aggregating the metric scores. Must have one entry per metric included 
-#'   in \code{df}.
-#' @param initialFlips,initialOffsets,initialTransforms Named vectors 
-#'   giving the initial value of the flip, offset and transforms for each 
-#'   metric. 
-#' @param initialCuts Named list giving the initial values of cut points used to
-#'   make each metric categorical.
-#' @param metricGroups Named list of named character vectors. Each list entry 
-#'   corresponds to one grouping of metrics. The grouping much be a named 
-#'   vector indicating the respective group for each metric. 
+#'   each metric to use for aggregating them into a final score. Must contain 
+#'   one entry per metric included in \code{metrics_num} or \code{metrics_cat}.
+#' @param initialTransforms Named list with initial values of transformation 
+#'   parameters for each metric. Each list entry should correspond to one 
+#'   metric, and take the form of a list with up to four elements, named:
+#'   \itemize{
+#'   \item \code{flip:} Logical scalar; whether or not to flip the sign of the 
+#'     metric values. Defaults to \code{FALSE}.
+#'   \item \code{offset:} Numeric scalar; offset to add to the (flipped) 
+#'     metric values. Defaults to \code{0}.
+#'   \item \code{transform:} Character scalar; one of 'None', 'z-score', 
+#'     '\[0,1\]', '\[-1,1\]' or 'Rank', indicating which transform to apply to 
+#'     the metric values (after any flipping and/or adding the offset). 
+#'     Defaults to 'None'.
+#'   \item \code{cuts:} Numeric vector or \code{NULL}; the cut points that will 
+#'     be used to bin the metric values (after the other transformations). 
+#'     Defaults to \code{NULL}. 
+#'   }
```

Comment:
Lazy me reminded me we are using the markdown formatting in the docs, shall I satisfy lazy me on this 😬 ?

Reply:
Oh, right - good point. I'll fix this.

Reply:
Fixed